packages/codebytes/src/codeByteEditor.tsx

```diff
@@ -13,7 +13,7 @@ import { trackUserImpression } from './libs/eventTracking';
 import { CodeByteEditorProps } from './types';
 
 const editorStates = states({
-  isIFrame: { height: '100vh' },
+  isIFrame: { height: '100vh', width: '100%' },
```

Comment:
any reason to use vh for height and not vw for width?

Reply:
i tried vw for width, but it's adding a weird scroll, and so opted to go with `100%` width

Reply:
gotcha all good